data-products/_example_2/application/transform.py

```diff
@@ -1,53 +1,32 @@
 import pandas as pd
-import os
-from sqlalchemy import create_engine
 import logging
-
+import boto3
 logging.basicConfig()
 
-# This is a temporary solution, see DLAB-33 and DLAB-34 for more details.
-# DB_ENDPOINT = os.environ.get("DB_ENDPOINT")
-# DB_USERNAME = os.environ.get("DB_USERNAME")
-# DB_PASSWORD = os.environ.get("DB_PASSWORD")
-# DB_NAME = os.environ.get("DB_NAME")
-# DB_PORT = os.environ.get("DB_PORT", 5432)
 
+def get_data(bucket: str, key: str) -> pd.DataFrame:
+    s3_client = boto3.client("s3")
```

Comment:
probably good practice to instantiate the client outside of the function and pass it in as a default.